### DISCUSSION TITLE
Cats module has native disabled?

### DISCUSSION THREAD

**Discussion by Quafadas** [outr/scribe#586] [General]
I was experimenting with compiling something to native, and I'm down to 

`not found: https://repo1.maven.org/maven2/com/outr/scribe-cats_native0.4_3/3.13.5/scribe-cats_native0.4_3-3.13.5.pom`

I can see it's quite deliberately commented out in the build file. 
https://github.com/outr/scribe/blob/25e17f8876cfdc62e1e8c776933175f828bd5a04/build.sbt#L145

Which makes it look like it was there once? There's no urgency. I guess I'm asking how deliberate / fundamental the choice to remove it was?

**Comment by darkfrog26** [comment]
When I did the last release of Scribe, `cats-effect` hadn't been updated to support ScalaNative 0.4 yet, but it looks like it has been now. I'll try to get this updated and a new release out soon.

**Comment by darkfrog26** [comment]
Sorry, I take that back. It's ScalaNative 0.5, and it doesn't support it yet. Since it looks like you're working with 0.4 you can just go back to an earlier release. It looks like `3.12.2` is using ScalaNative 0.4.

**Reply by Quafadas** [reply]
Yeah - I think that would work... I'll o with that... 

And the commentary all makes sense, thanks for writing back.
